src/modules/workspace/create.ts

```diff
@@ -1,9 +1,9 @@
 import { Workspaces } from '@vtex/api'
 import chalk from 'chalk'
-import { CommandError } from '../../api/error/errors'
+import { createFlowIssueError } from '../../api/error'
+import { ErrorReport } from '../../api/error/ErrorReport'
```

Comment:
Would it be possible to import like:
```suggestion
import { createFlowIssueError, ErrorReport } from '../../api/error'
```
?

Reply:
Yes it is. I used to import `ErrorReport` and `ErrorKind` as `import {ErrorReport, ErrorKind} from '../../api/error` but this can cause a ciclyc  import, so to solve this problem the correct way is to import from separeted paths, so I'm going to fix this to the full path